fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/InstrumentedReadTransaction.java

```diff
@@ -199,16 +209,6 @@ public Executor getExecutor() {
         return underlying.getExecutor();
     }
 
-    @Override
-    public CompletableFuture<Long> getEstimatedRangeSizeBytes(final byte[] begin, final byte[] end) {
```

Comment:
This is moving these two methods to I think better align them with where these methods are in the interface.